public_dropin_environments/r_lang/start_server.sh

```diff
@@ -2,7 +2,7 @@
 cd /opt/code/ || exit 1
 export PYTHONPATH=/opt/code
 
-CMD="drum server -cd . --address 0.0.0.0:8080 --with-error-server"
+CMD="drum server -cd . --address 0.0.0.0:8080 --production --max-workers 1 --show-stacktrace"
```

Comment:
do we no longer need the error server?

Reply:
--with-error-server param has been removed. Error server has been used to serve error messages, if model loading fails.
With uwsgi server, loading fails on worker, so main drum process won't fail. Instead worker will be serving error message.